test/testcafe/spec/TerminalView_spec.js

```diff
@@ -149,8 +154,9 @@ async function setupOtpOnly(t) {
 
 // Make sure geolocation is only displayed on OTP Only page when present in response
 [
-  ['Should have entry for geolocation on OTP info page when accessed', terminalReturnOtpOnlyMock, true],
-  ['Should not have entry for geolocation on OTP info page (w/out location) when accessed', terminalReturnOtpOnlyNoLocationMock, false],
+  ['Should have entry for geolocation on OTP info page when accessed', terminalReturnOtpOnlyMock, 'Toronto, Ontario, Canada'],
```

Comment:
Nice work adding these tests.. But i would suggest to add these test to specific view. for eg in this it should be `EmailMagicLinkOTPTerminalView.js`.. 

Keeping TerminalView spec clean can help other devs who are not aware of email magic link related tests.

Reply:
Totally agree, just changed it